pints/_log_likelihoods.py

```diff
@@ -578,57 +578,78 @@ class MultiplicativeGaussianLogLikelihood(pints.ProblemLogLikelihood):
     .. math::
         v(t) \sim \text{ iid } N(0, \sigma)
 
-    Note that the scalar parameter ``eta`` controls the exponential dependence
-    of the noise on the function output, while the scalar parameter ``sigma``
-    provides a baseline level of the noise standard deviation. This model leads
-    to a log likelihood of
+    Note that the scalar parameter :math:`\eta` controls the exponential
+    dependence of the noise on the function output, while the scalar parameter
+    :math:`\sigma` provides a baseline level of the noise standard deviation.
+    This model leads to a log likelihood of
 
     .. math::
-        \log{L(\theta, \sigma, \eta | \boldsymbol{x})} =
+        \log{L(\theta, \eta , \sigma | X^{\text{obs}})} =
             -\frac{n_t}{2} \log{2 \pi}
             -\sum_{i=1}^{n_t}{\log{f(t_i, \theta)^\eta \sigma}}
             -\frac{1}{2}\sum_{i=1}^{n_t}
-                \frac{(X(t_i) - f(t_i, \theta))^2}
+                \frac{(X^{\text{obs}}_{i} - f(t_i, \theta))^2}
                 {(f(t_i, \theta)^\eta \sigma)^2}
 
-    where ``n_t`` is the number of time points in the series.
+    where :math:`n_t` is the number of time points in the series, and
+    :math:`X^{\text{obs}}_{i}` the measurement at time :math:`t_i`.
+
+    For a system with :math:`n_o` outputs, this becomes
+
+    .. math::
+        \log{L(\theta, \eta , \sigma | \boldsymbol{x})} =
+            -\frac{n_tn_o}{2} \log{2 \pi}
+            -\sum ^{n_o}_{j=1}\sum_{i=1}^{n_t}{\log{f_j(t_i, \theta)^\eta
+            \sigma _j}}
+            -\sum ^{n_o}_{j=1}\sum_{i=1}^{n_t}
```

Comment:
Please put the 1/2 in front of the sum again, to match the notation above!

Reply:
done